docs/SqlBindingRelatedRepos.md

```diff
@@ -0,0 +1,35 @@
+# Azure SQL Bindings for Azure Functions 
+ 
```

Comment:
Would suggest adding a brief one or two sentence description of what this document is